### ISSUE TITLE
Cannot import vary the URL has changed

### ISSUE THREAD

**Issue by Ciantic** [asos-craigmorten/opine#48]
- Deno Version: 1.2.2
- v8 Version: 8.5.216
- Typescript Version: 3.9.2
- Opine Version: 0.19.1

```shell
error: Import 'https://deno.land/x/vary@dc17535/mod.ts' failed: 403 Forbidden
Imported from "https://deno.land/x/opine@0.19.1/deps.ts:46"
```

This is caused by this line:
https://github.com/asos-craigmorten/opine/blob/main/deps.ts#L46

Not sure what the version is supposed to be, I think it should be:

https://deno.land/x/vary@1.0.0/mod.ts

**Comment by asos-craigmorten** [comment]
This is / was caused by a breaking change to the Deno registry today meaning we can no longer reference commits and/or branches - see https://deno.land/posts/registry2

I believe at the time of using this module it didn’t have a tagged release hence the commit, hopefully the fix is as simple as you say and can use `1.0.0` (or later version if exists).

**Comment by asos-craigmorten** [comment]
@Ciantic Please try out `opine@0.20.0` - should hopefully be resolved 😄
